### PR TITLE
added ipv6 support

### PR DESCRIPTION
**Pull request by gdsoumya**
## Proposed changes

This PR adds ipv6 support to litmus portal

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [x] New feature (non-breaking change which adds functionality)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: litmus-portal/graphql-server/pkg/k8s/cluster.go [hunk 1]
```diff
@@ -153,7 +151,7 @@ func GetServerEndpoint() (string, error) {
 			Scheme = "http"
 		}
 
-		FinalUrl = Scheme + "://" + IPAddress + "/" + IngressPath
+		FinalUrl = Scheme + "://" + wrapIPV6(IPAddress) + "/" + IngressPath
```
Comment (rajdas98) [inline litmus-portal/graphql-server/pkg/k8s/cluster.go:154]: Should we give an option between IPV6 and IPV4 by a flag?

**Review by rajdas98 — CHANGES_REQUESTED** [review]

File: litmus-portal/graphql-server/pkg/k8s/cluster.go (hunk 1 above)
Comment (rajdas98) [reply litmus-portal/graphql-server/pkg/k8s/cluster.go:154]: I can see a function, will it automatically detect?

Comment (gdsoumya) [reply litmus-portal/graphql-server/pkg/k8s/cluster.go:154]: Yes auto detect

File: litmus-portal/cluster-k8s-manifest.yml [hunk 2]
```diff
@@ -695,6 +695,7 @@ spec:
       containers:
         - name: mongo
           image: litmuschaos/mongo:4.2.8
+          args: ["--ipv6"]
```
Comment (Jonsy13) [inline litmus-portal/cluster-k8s-manifest.yml:698]: Will it work for ipv4 with this ipv6 flag? @gdsoumya

Comment (gdsoumya) [reply litmus-portal/cluster-k8s-manifest.yml:698]: works for both

**Review by Jonsy13 — APPROVED** [review]

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**